docs/API_README.md

```diff
@@ -0,0 +1,142 @@
+## API Docs
+
+**Interface to sign in**
+
+| URI      | Method | Authorization |
+|----------|--------|---------------|
+|`/token/` | `POST` | not required  |
+
+
+Params:
+ * `email=[string]`
+ * `password=[string]`
+
+Success Response:
+
+* Code: 200
+* Content: `{"refresh": "<refresh token>", "access": "<access token>"}`, where `<refresh token>` and `<access token>` are the valid JWT tokens.
+
+Specific Error Response
+* Code: 400
+* Content: `{"detail": "No active account found with the given credentials"}`
+* How to solve: register the user or provide the valid credentials.
```

Comment:
I guess only registering a user won't solve the problem, so I would paraphrase it:

register a user by yourself or ask your sysadmin to do it **and** provide valid credentials.